File: CursoPOOUber/PHP/UberVan.php
<?php
require_once('Car.php');
class UberVan extends Car{
    public $typeCarAccepted;
    public $material;

    public function __construct($license, Account $driver, $typeCarAccepted, $material)
    {
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->material = $material;
    }
}

?>

File: CursoPOOUber/Java/UberVan.java
class UberVan extends Car {
    String typeCarAccepted;
    String seatsMaterial;
    private Integer passenger;

    public UberVan(String license, Account driver, String typeCarAccepted, String seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    void printDataVan() {
        System.out.println("License: " + license + " Name Driver: " + driver.name + " Tipo de auto: " 
        + typeCarAccepted + " Material de los asientos: " + seatsMaterial);
    }
    
    @Override
    public void setPassenger(Integer passenger) {
        if (passenger == 6) {
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }
}